lib/Photo/Photo.h

```diff
@@ -0,0 +1,29 @@
+#ifndef PHOTO_H
+#define PHOTO_H
+#include "Arduino.h"
+#include "motors.h"
+#include "constants.h"
+
+class Photo {
+public:
+    Photo();
+    int ReadPhotoLeft();
+    int ReadPhotoRight();
+    int ReadPhotoFront();
+    bool CheckPhotoLeft();
+    bool CheckPhotoRight();
+    bool CheckPhotoFront();
+
+private:
+    // Photo Left
+    int photo_left[kPhotoLeftElements];
```

Comment:
whats the max value possible for each? if the max is less or equal to 255 use uint8 (uint8_t y think) instead it would take 1/4 of the space used instead